### DISCUSSION TITLE
Chrome->Firefox License

### DISCUSSION THREAD

**Discussion by arifarifoglu** [aclap-dev/video-downloadhelper#1059] [Help and Q&A]
### Your question

Dear Support Team,

I purchased the VideoDownloader Helper license for the Chrome version .However, I recently discovered that the Chrome version does not allow me to download YouTube videos. I would like to know if there is any way to convert my Chrome license to use it on Mozilla Firefox or if there is still a possibility of a refund so that I can purchase the Mozilla Firefox version.

I look forward to your response and thank you in advance for your attention.

### Addon & CoApp version and details

My personal registration code is: ***********************52bddc93243c6e

**Comment by GrampaWildWilly** [comment]
Take a look at #1054.

**Comment by paulrouget** [comment]
https://github.com/aclap-dev/video-downloadhelper/wiki/I-need-help#license-problem
